unskript-ctl/unskript_ctl_notification.py

```diff
@@ -742,4 +754,8 @@ def _send_email(self, **kwargs):
                         region = kwargs.get('region', aws.get('region')),
                         subject = kwargs.get('subject', self.email_config.get('email_subject_line', 'Run Result'))
                         ) 
+        if retval:
+            print("Notification successuflly sent")
```

Comment:
successfully. It will be better to print for each type enabled, if it was sent successfully or not

Reply:
like slack, email

Reply:
Took care of the Comments in the latest  push